### ISSUE TITLE
Regression: Quarantine flag resurfaces (macOS)

### ISSUE THREAD

**Issue by corwin-of-amber** [nwjs/nw.js#7646]
NWJS Version : 0.50.2
Operating System : 10.15.7

### Expected behavior

Files created by NWjs (or a subprocess) should *not* have their quarantine flag set.

### Actual behavior

Files are quarantined. For example, I ran a shell from within NWjs, and sent it the following commands:
```
gcc hello.c
./a.out
```

### How to reproduce

There is really no need to go through the tedious process of creating an executable from within NWjs to see this bug. Suffices to look in `nwjs.app/Contents/Info.plist` to see that:
```
	<key>LSFileQuarantineEnabled</key>
	<true/>
```

This issue has actually been solved in the past by @rogerwang (#2294), and I am a bit perplexed as the source files referenced there — src/mac/app-Info.plist and src/mac/helper-Info.plist — in the repo still bear the value `<false/>`.
However when downloading the release binaries (https://dl.nwjs.io/v0.50.2/nwjs-v0.50.2-osx-x64.zip, as https://github.com/nwjs/npm-installer does), the `Info.plist` does not look like either one and the flag is in fact `<true/>`.

**Comment by corwin-of-amber** [comment]
Encountered this again from 0.58.

**Comment by corwin-of-amber** [comment]
Yes. Still.

**Comment by ayushmanchhabra** [comment]
So the fix is to have LSFileQuarantineEnabled set to false in the Contents/Info.plist and Helper Apps Info.plist?

**Comment by corwin-of-amber** [comment]
Yes. Except, it is already `false` in the source tree (https://github.com/nwjs/nw.js/blob/aa5dc8fa9fc2fe8fa31591bd2e90cccd4939df44/src/mac/app-Info.plist#L27), so I have no idea what makes it become `true` in the installed `nwjs.app` (generated by e.g. `npm i -g nwjs@sdk`).

**Comment by ayushmanchhabra** [comment]
I guess I can add a step in nw-builder to set it to false.

**Comment by ayushmanchhabra** [comment]
I have published a [workaround/fix](https://github.com/nwutils/nw-builder/pull/1631) for this in nw-builder.

**Comment by ayushmanchhabra** [comment]
@corwin-of-amber I don't know if you use nw-builder, but could you check the latest release? I don't have a Mac.

**Comment by corwin-of-amber** [comment]
@ayushmanchhabra I have not used nw-builder before but I will try. Thanks!

**Comment by ayushmanchhabra** [comment]
So this flag basically forces a security check/scan on MacOS everytime the application runs? For example, if its signature is not verified or not downloaded from App Store, it will get flagged by Gatekeeper?

**Comment by ayushmanchhabra** [comment]
https://github.com/nwutils/nw-builder/pull/1634
The `LSFileQuarantineEnabled` now defaults to `false` in `nw-builder`, but can be toggled on by setting `options.app.LSFileQuarantineEnabled` to `true`.
